src/components/directorybrowser/directorybrowser.js

```diff
@@ -70,76 +167,142 @@ define(["loading", "dialogHelper", "dom", "listViewStyle", "emby-input", "emby-b
             }
         }).catch(function(response) {
             if (response) {
-                if (404 === response.status) return alertText("The path could not be found. Please ensure the path is valid and try again."), Promise.reject();
-                if (500 === response.status) return alertText(validateWriteable ? "Jellyfin Server requires write access to this folder. Please ensure write access and try again." : "The path could not be found. Please ensure the path is valid and try again."), Promise.reject()
+                if (response.status === 404) {
+                    alertText("The path could not be found. Please ensure the path is valid and try again.");
```

Comment:
Let's add a `TODO` on adding localization here and below

Reply:
It might be a good idea to make a helper function for alerts that takes a key from the globalization .json files and alerts that. Then we can change it everywhere.